src/components/Header/HeaderPopover.tsx

```diff
@@ -1,10 +1,24 @@
+// @ts-nocheck
```

Comment:
instead of skipping type checking entirely, can we just add `@ts-ignore` wherever we are getting type errors related to `react-spring`?

Reply:
As I said in the comment below, in this file we have more types problem. But we need to update the lib version so the types have effect.